templates/snmptrapd.conf.erb

```diff
@@ -21,16 +21,22 @@ disableAuthorization <%= @disable_authorization %>
 ################################################################################
 # NOTIFICATION PROCESSING
 
-<% @trap_handlers.each do |handler| -%>
+<% if not @trap_handlers.nil? -%>
```

Comment:
you're probably not interested in turning this into an epp template, are you? :D 

totally not required, but always nice to see.

Reply:
Not in the slightest.. Unfortunately I don't see any value in doing that conversion.

Reply:
My goal with this PR was to deprecate params.pp in favor of data in module. Part of that included adding data types to all the parameters. I want to keep this PR as small as possible to reach that goal.